### PR TITLE
[FUSEQE-8681][pf4-migration] Extension page improvement

### PR DESCRIPTION
**Pull request by mkralik3**

### CODE REVIEW

File: ui-tests/src/test/resources/features/integrations/amq-to-rest.feature [hunk 1]
```diff
@@ -20,11 +20,8 @@ Feature: Integration - AMQ to REST
     And Set Todo app credentials
     Then deploy ActiveMQ broker
 
-    When click on the "Customizations" link
-    And navigate to the "Extensions" page
-    And click on the "Import Extension" link
-    And upload extension with name "sample-damage-reporter-extension-1.4.8.jar" from relative path "./src/test/resources/extensions/"
-    Then click on the "Import Extension" button
+    When import extensions
+      | sample-damage-reporter-extension-1.4.8.jar | ./src/test/resources/extensions/ |
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/amq-to-rest.feature:24]: Great change!

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - DB to DB | Read and update operations 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**